singularity-opac/neutrinos/s_opac_neutrinos.hpp

```diff
@@ -25,7 +25,6 @@
 namespace singularity {
 namespace neutrinos {
 
-// TODO(JMM): Include chemical potential
```

Comment:
I removed this previously, but it didnt get committed into an earlier PR, oops (too many computers)